GameData/RealismOverhaul/RO_SuggestedMods/Squad/RO_Squad_Communication.cfg

```diff
@@ -254,6 +255,54 @@
     }
 }
 
+@PART[HighGainAntenna5_v2]:FOR[RealismOverhaul]:NEEDS[RealAntennas]
+{
+    %MODULE[ModuleRealAntenna] { %antennaDiameter = 0.5 }
+    @mass = 0.001
+}
+@PART[HighGainAntenna5_v2]:AFTER[RemoteTech]
+{
+    @description ^=:$: Wide angle and reasonable bandwidth among the inner planets. <b><color=#00eaf0>Effective range to Earth DSN [400 Gm] — Power Consumption [30 Watts] — Maximum Data Rate [1 Mbit/s] — Use Case [Mars, Vesta (not at Conjunction), Ceres (not at Conjunction)]</color></b>
+
+    @MODULE[ModuleRTAntenna]
+    {
+        @Mode0DishRange = 0
+        @Mode1DishRange = 1.4e9
+        @EnergyCost = 0.03
+        %MaxQ = 6000
+        @DishAngle = 7.0
+        %DeployFxModules = 0
+        %ProgressFxModules = 1
+
+        @TRANSMITTER
+        {
+            @PacketInterval = 1.0
+            @PacketSize = 1.024
+            @PacketResourceCost = 0.01
+        }
+    }
+}
+
+@PART[HighGainAntenna5]:FOR[RealismOverhaul]
+{
+    %RSSROConfig = True
+    %deprecated = True
```

Comment:
And now, setting %deprecated = True is the *only* thing this patch on HighGainAntenna5 should be doing.